spec/requests/admin/firms_spec.rb

```diff
@@ -3,9 +3,13 @@
 module Admin
   RSpec.describe FirmsController, type: :request do
     let(:admin_user) { create :admin_user }
+    let(:permission) { create :permission }
     let!(:firms) { create_list :firm, 3 }
 
-    before { sign_in admin_user }
+    before do
+      firms.first.update!({ permission_ids: [permission.id] })
```

Comment:
Reviewing this after #1782 should we update the factory to have a default permission too?

Reply:
Good point. I've updated the factory so that it creates a permission by default, and fixed some tests.